packages/db/scripts/embeddings-indexes/add_embedding_indexes.sql

```diff
@@ -0,0 +1,19 @@
+-- Add vector index for quiz_questions table
```

Comment:
Running this should add the ivfflat indexes that optimise cosine similarity queries. We would need to run this manually or whenever we run Prisma migrate. But I think this would lead to getting schema drift warnings.